packages/types/src/http/sales-channel/admin/sales-channel.ts

```diff
@@ -0,0 +1,26 @@
+import { PaginatedResponse } from "../../common"
+
+/**
+ * @experimental
+ */
+interface SalesChannelResponse {
+  id: string
+  name: string
+  description: string | null
+  is_disabled: boolean
+  metadata: Record<string, unknown> | null
+}
+
+/**
+ * @experimental
+ */
+export interface AdminSalesChannelListResponse extends PaginatedResponse {
```

Comment:
@adrien2p, think we should structure the responses this way instead. Minor difference from what we have right now

Reply:
yeah that sounds good 👍